pkg/piecestore/rpc/server/server_test.go

```diff
@@ -115,6 +115,9 @@ func TestPiece(t *testing.T) {
 					//TODO (windows): ignoring for windows due to different underlying error
 					return
 				}
+
+				fmt.Printf("#### expected: %+v\n #### actual: %+v\n", tt.err, err)
```

Comment:
do we need to keep this?

Reply:
it's still WIP, but this will be removed for sure

Reply:
gotcha